test-infra/camel-test-infra-jetty/pom.xml

```diff
@@ -31,6 +31,11 @@
     <name>Camel :: Test Infra :: Jetty</name>
 
     <dependencies>
+        <dependency>
```

Comment:
Is there a reason for this change

Reply:
Same reason as documented here
https://github.com/tdiesler/camel/blob/CAMEL-22032/dsl/camel-jbang/camel-jbang-plugin-kubernetes/pom.xml#L87

IntelliJ seems to handle test-jar dependencies different different from maven.

Reply:
ok